compiler-rt/lib/builtins/os_version_check.c

```diff
@@ -282,6 +285,29 @@ int32_t __isPlatformVersionAtLeast(uint32_t Platform, uint32_t Major,
   return AvailabilityVersionCheck(1, Versions);
 }
 
+#if TARGET_OS_OSX
+
+int32_t __isPlatformOrVariantPlatformVersionAtLeast(
+    uint32_t Platform, uint32_t Major, uint32_t Minor, uint32_t Subminor,
+    uint32_t Platform2, uint32_t Major2, uint32_t Minor2, uint32_t Subminor2) {
```

Comment:
Can you document what the `Platform` arguments are supposed to be? Where do those integer values come from? It seems like they should be the same as for `__isPlatformVersionAtLeast`, but that isn't documented either so these constants seem to come out of nowhere.